quesma/testdata/dashboard-1/aggregation_requests.go

```diff
@@ -375,89 +357,75 @@ var AggregationTests = []testdata.AggregationTestCase{
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 1.0),
-					model.NewQueryResultCol("quantile_95", []float64{77}),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957330000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 3.0),
-					model.NewQueryResultCol("quantile_95", []float64{71}),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 1.0),
-					model.NewQueryResultCol("quantile_95", []float64{80.44999999999999}),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 3.0),
-					model.NewQueryResultCol("quantile_95", []float64{63}),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 5.0),
-					model.NewQueryResultCol("quantile_95", []float64{83.8}),
-				}},
-			},
-			{
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957330000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 1.0),
-					model.NewQueryResultCol("doc_count", 159),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957330000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 3.0),
-					model.NewQueryResultCol("doc_count", 8),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 1.0),
-					model.NewQueryResultCol("doc_count", 52),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 3.0),
-					model.NewQueryResultCol("doc_count", 21),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol(`floor("billingRegion\" / 0.020000) * 0.020000`, 5.0),
-					model.NewQueryResultCol("doc_count", 5),
-				}},
-			},
-			{
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957330000/30000)),
-					model.NewQueryResultCol("doc_count", 167),
-				}},
-				{Cols: []model.QueryResultCol{
-					model.NewQueryResultCol("key", int64(1713957360000/30000)),
-					model.NewQueryResultCol("doc_count", 78),
-				}},
-			},
-		},
+		/*
+			ExpectedResults: [][]model.QueryResultRow{
+				{{Cols: []model.QueryResultCol{model.NewQueryResultCol("hits", uint64(4800))}}},
+				{
+					{Cols: []model.QueryResultCol{
```

Comment:
why remove everywhere, but leave here as comment?

Reply:
This test does not have pancake version, so comment might be useful to build pancake equivalent.